setup.cfg

```diff
@@ -15,6 +15,7 @@ per-file-ignores =
     examples/torch/*:N812
     src/garage/torch/*:N812,D403
     tests/garage/torch/*:N812,D
+    tests/integration_tests/test_examples.py:D
```

Comment:
integration tests require documentation because they are complicated. please remove.

Reply:
Yeah please fix whatever's wrong with the file. It should be fine already.

Reply:
fixed
\